rust/kernel/cpu.rs

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! CPU utilities for SMP systems.
+//!
+//! C header: [`include/linux/smp.h`](../../../../include/linux/smp.h).
+//!
+//! Note: This module is a work in progress.
+
+use crate::{bindings, error::code::*, prelude::*, to_result};
+use core::{marker::PhantomData, ops::Deref};
+
+/// Represents an SMP CPU by its zero-based kernel processor id.
+///
+/// Cpu values can be safely passed between threads (and across physical CPUs) as they are
```

Comment:
I think you are referring to the type itself, so:

```suggestion
/// [`Cpu`] values can be safely passed between threads (and across physical CPUs) as they are
```

Otherwise:

```suggestion
/// CPU values can be safely passed between threads (and across physical CPUs) as they are
```